src/tests/Microsoft.PowerFx.Core.Tests/BindingEngineTests.cs

```diff
@@ -143,6 +143,28 @@ public void CheckBindError()
 
             Assert.False(result.IsSuccess);
             AssertContainsError(result, "Error 2-5: Name isn't valid. 'foo' isn't recognized");
+        }
+
+        [Theory]
+
+        // Runtime errors

```

Comment:
>Runtime [](http://example.com/codeflow?start=11&length=7)

They are not really runtime errors, they are binding errors.

Reply:
To be addressed in #812